shared/chat/conversation/info-panel/index.tsx

```diff
@@ -124,11 +124,7 @@ const TabText = ({selected, text}: {selected: boolean; text: string}) => (
   </Kb.Text>
 )
 
-class _InfoPanel extends React.Component<InfoPanelProps> {
-  componentDidMount() {
-    this._retryLoad()
```

Comment:
done

Reply:
A barrage of incoming `chat.1.chatUi.chatLoadGalleryHit` calls makes this stutter in any ad-hoc chats I've used significantly. I think if this only happened the first time it'd be ok, but it happens every time. Ignoring any protocol changes that could make this better, for now delaying this for the timeout of the animation seems like it'd make this better.

Reply:
I'll do a better fix. The way this works isn't great

Reply:
Can you just start the gallery load when the animation finishes?

Reply:
yeah i'm just doing a simple timeout. how this whole thing works is mostly gross. the parent is managing everything, for no real reason, so instead of making that even more true, doing the simple thing for now